### ISSUE TITLE
Is there a way to have multiple package lists in adition to [packages] and [dev-packages] in our Pipfile?

### ISSUE THREAD

**Issue by joaqo** [pypa/pipenv#2151]
Been searching all around but my google skills are failing me.

I have a project in which I have a client and a server which have different package requirements, I'd like to be able to have them be separate in Pipfile (like its done in [packages] vs [dev-packages]), or maybe have different Pipfiles, not sure how this is usually solved in pipenv. Normally I would have just created several requirements.txt files, but I really really like pipenv's Pipfiles!

So ideally I would be able to do `pipenv install --server` or pipenv install --client` much in the same way I can do `pipenv install`/`pipenv install --dev`.

Any suggestions?

**Comment by techalchemy** [comment]
FYI Kenneth has already rejected this a number of times. It’s going to take a persuasive use case to move him (and me for that matter) on this point. Too many knobs is likely to cause more harm than good.

**Comment by joaqo** [comment]
Hey @uranusjr and @techalchemy thanks a lot for the fast answer! Is there a currently recommended way to handle this while the feature gets added? I am guessing its just probably using multiple requirements.txt files?

**Comment by uranusjr** [comment]
Yes, and you can set up some [scripts](https://docs.pipenv.org/advanced/#custom-script-shortcuts) to help improving the setup experience:

```toml
[scripts]
install-client = "pip install client-requirements.txt"
install-server = "pip install server-requirements.txt"
```

**Comment by joaqo** [comment]
Great, I really like that as a temp solution, thanks!
